### PR TITLE
Translated how-to-catch-outgoing-emails-locally.md into ukrainian

### PR DESCRIPTION
**Pull request by philei-tt**
translated "how-to-catch-outgoing-emails-locally.md" into ukrainian

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.
This is a part of Ukrainian translation project started with a Feature Request here: https://github.com/freeCodeCamp/freeCodeCamp/issues/37228

### CODE REVIEW

**Review by last-genius — CHANGES_REQUESTED** [review]
Change "сріншот" to "скріншот"

**Review by last-genius — APPROVED** [review]

**Comment by JanusDG** [comment]
This is the startpoint of my https://github.com/freeCodeCamp/freeCodeCamp/pull/37233

File: docs/i18n-languages/ukrainian/how-to-catch-outgoing-emails-locally.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+|-|
+<!-- do not translate this -->
+
+# Як локально відловлювати отримання вихідних електронних листів без їх реальної відправки (Потрібно тільки для робочих процесів електроної пошти)
+
+> **Замітка:** Це не **обов'язковий** крок - потрібно тільки при роботі з робочими процесами електронної пошти
+
+## Вступ
+
+Деякі робочі процеси електронної пошти (наприклад оновлення електронної пошти користувача) вимагають щоб внутрішній сервер через api відправляв електронні листи. Під час розробки ви можете використовувати інструмент для локального відловлення електронної пошти, замість того щоб використовувати реального постачальника електронної пошти та відправляти реальні електронні листи. MailHog - це один з таких засобів тестування електронної пошти для розробників, який ловить електронні листи, надіслані вашим локальним екземляром freeCodeCamp.
```
Comment (yevhenorlov) [inline docs/i18n-languages/ukrainian/how-to-catch-outgoing-emails-locally.md]: change "екземляром" to "екземпляром"

File: docs/i18n-languages/ukrainian/how-to-catch-outgoing-emails-locally.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+## Установка MailHog
+
+MailHog може бути встановлений на macOS, Windows and Linux.
+
+- [Установка MailHog on macOS](#installing-mailhog-on-macos)
+- [Установка MailHog on Windows](#installing-mailhog-on-windows)
+- [Установка MailHog on Linux](#installing-mailhog-on-linux)
```
Comment (yevhenorlov) [inline docs/i18n-languages/ukrainian/how-to-catch-outgoing-emails-locally.md]: maybe change "on" to "на"?

File: docs/i18n-languages/ukrainian/how-to-catch-outgoing-emails-locally.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+Як встановити MailHog на macOS з допомогою [Homebrew](https://brew.sh/):
+
+```bash
+brew install mailhog
+brew services start mailhog
+```
+
+Дані команди встановлять і запустять службу MailHog у фоновому режимі.
+
+Після цього ви можете перейти безпосередньо до  [використання MailHog](#using-mailhog).
```
Comment (yevhenorlov) [inline docs/i18n-languages/ukrainian/how-to-catch-outgoing-emails-locally.md]: double space after "до"

File: docs/i18n-languages/ukrainian/how-to-catch-outgoing-emails-locally.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+### Установка MailHog on Windows
+
+Завантажте останню версію MailHog з  [офіційного репозиторію](https://github.com/mailhog/MailHog/releases). Виберіть посилання для вашої версії Windows (32 або 64 біт) і .exe файл буде завантажений на ваш комп'ютер.
```
Comment (yevhenorlov) [inline docs/i18n-languages/ukrainian/how-to-catch-outgoing-emails-locally.md]: double space after "з"

File: docs/i18n-languages/ukrainian/how-to-catch-outgoing-emails-locally.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+Щоб закрити MailHog, закрийте командний рядок. Щоб запустити його знову, клікніть той же .exe файл. Вам не потрібно завантажувати кожен раз новий файл.
+
+Потім ви можете перейти беспосередньо до [використання MailHog](#using-mailhog).
+
+### Установка MailHog on Linux
+
+Спочатку встановіть [Go](https://golang.org).
+
+Для систем на базі Debian, таких як Ubuntu і Linux Mint, виконайте команду
```
Comment (yevhenorlov) [inline docs/i18n-languages/ukrainian/how-to-catch-outgoing-emails-locally.md]: let's add colon here for consistency with line 55

File: docs/i18n-languages/ukrainian/how-to-catch-outgoing-emails-locally.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+Відкрийте електронний лист, і ви побачите дві вкладки, де ви можете переглянути вміст: звичайний текст і джерело. Переконайтеся, що ви перебуваєте на вкладці звичайного тексту.
+
+![MailHog Screenshot 3](images/mailhog/3.jpg)
+
+Будь-які посилання в листі так само повинні бути доступні для перегляду.
+
+## Корисні посилання
+
+- З будь-якими іншими питаннями, пов'язаних з MailHog або інструкціями по призначених для користувача налаштувань, ви можете ознайомитися в [MailHog](https://github.com/mailhog/MailHog)
```
Comment (yevhenorlov) [inline docs/i18n-languages/ukrainian/how-to-catch-outgoing-emails-locally.md]: "пов'язаних" -> "пов'язаними", 
"налаштувань" -> "налаштуваннях"

**Review by yevhenorlov — CHANGES_REQUESTED** [review]
well done! please take a look at the comments

**Comment by raisedadead** [comment]
Closing stale, there are review comments on this thread that have not been addressed? Please feel free to make a fresh PR if still interested.

**Pull request closed without merging**